crates/llama-cpp-bindings/build.rs

```diff
@@ -69,7 +69,14 @@ fn main() {
     }
 
     let dst = config.build();
-    println!("cargo:rustc-link-search=native={}/build", dst.display());
+    if cfg!(target_os = "windows") {
+        #[cfg(debug_assertions)]
```

Comment:
`debug_assertions` is turned on when `debug-assertions` settiongs from cargo is true.
The default profile for `cargo build` is:
```
[profile.dev]
......
debug-assertions = true
```

So this help differentiate between `debug` mode or `release` mode.